.github/workflows/e2e-tests.yml

```diff
@@ -168,6 +167,58 @@ jobs:
             exit 1
           fi
 
+          echo "Sending second job creation request..."
+          job_response=$(http --check-status --ignore-stdin --timeout=10 --print=b -A bearer -a "${tml_token}" POST localhost:8080/api/v1/job/queue \
+            supervisor_id="${tml_supervisor}" \
+            job_request[request_id]="$(uuidgen)" \
+            job_request[init_spec][type]=image \
+            job_request[init_spec][image_id]="${tml_image_id}" \
+            job_request[ssh_keys]:='[]' \
+            job_request[restart_policy][remaining_restart_count]:=0 \
+            job_request[ssh_rendezvous_servers]:='[]' \
+            job_request[parameters]:='{}' \
+            job_request[tag_config]='' \
```

Comment:
I'm pretty sure that this request payload needs to be updated (similar to the one above), but we can try merging and seeing whether the queue catches this.

Reply:
Nevermind, the status checks on the last commit need to pass before GitHub even attempts this merge. If you could rebase these changes and also adjust this payload, this should be good to go.